auth_brute_force/controllers/controllers.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 GRAP - Sylvain LE GAL
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import fields, http, registry, SUPERUSER_ID
+from odoo.api import Environment
+from odoo.http import request
+from odoo.addons.web.controllers.main import Home, ensure_db
+
+_logger = logging.getLogger(__name__)
+
+
+class LoginController(Home):
+    @http.route()
+    def web_login(self, redirect=None, **kw):
+        if request.httprequest.method == 'POST':
+            ensure_db()
+            remote = request.httprequest.remote_addr
+            # Get registry and cursor
+            with registry(request.session.db).cursor() as cursor:
+                env = Environment(cursor, SUPERUSER_ID, {})
+                config_obj = env['ir.config_parameter']
+                attempt_obj = env['res.authentication.attempt']
+                banned_remote_obj = env['res.banned.remote']
+                # Get Settings
+                max_attempts_qty = int(config_obj.search_read(
```

Comment:
Use `config_obj.get_param('auth_brute_force.max_attempt_qty')` instead of `search_read`.